src/main/java/com/faforever/client/game/GameService.java

```diff
@@ -268,7 +290,7 @@ public ReadOnlyBooleanProperty gameRunningProperty() {
   /**
    * @param path a replay file that is readable by the preferences without any further conversion
    */
-  
+
```

Comment:
No empty line here

Reply:
Yeah I know but i deleted the empty lines in this class in 3 other PRs as well so i did not want to do it again. Because of one of the other 3 gets merged it will eliminate the empty lines as well

Reply:
But i am gonna do it a forth time ;)

Reply:
oh ok i did it but missed that one line